webextensions/manifest/theme.json

```diff
@@ -342,7 +342,7 @@
                   "version_added": false
                 },
                 "firefox": {
-                  "version_added": false
+                  "version_added": 63
```

Comment:
As for https://github.com/mdn/browser-compat-data/pull/2816/, this and line 273 need to be strings.